tests/nnstreamer_trainer/unittest_trainer.cc

```diff
@@ -73,7 +73,7 @@ TEST (tensor_trainer, SetParams)
       "gst-launch-1.0 datareposrc location=%s json=%s "
       "start-sample-index=3 stop-sample-index=202 tensors-sequence=0,1 epochs=1 ! "
       "tensor_trainer name=tensor_trainer framework=nntrainer model-config=%s "
-      "model-save-path=model.bin num-inputs=1 num-labels=1 "
+      "model-save-path=new_model.bin model_load_path=old_model.bin num-inputs=1 num-labels=1 "
```

Comment:
Does `set_property` treat `model_load_path` (with _) and `model-load-path` (with -) same?
Since this test passed, it seems to be treated as same, but just for check cause I didn't know it before

Reply:
How about adding what model_load_path property means in comment like other properties?

Reply:
Thank you, I'm sorry , I will fix typo.